libcloudforensics/providers/gcp/internal/storagetransfer.py

```diff
@@ -0,0 +1,160 @@
+# -*- coding: utf-8 -*-
+# Copyright 2021 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Google Cloud Storage Transfer functionalities."""
+
+from typing import TYPE_CHECKING, Dict, Any, Optional
+import datetime
+import time
+
+from libcloudforensics import errors
+from libcloudforensics import logging_utils
+from libcloudforensics.providers.aws.internal import account
+from libcloudforensics.providers.gcp.internal import common
+from libcloudforensics.providers.gcp.internal import storage as gcp_storage
+
+logging_utils.SetUpLogger(__name__)
+logger = logging_utils.GetLogger(__name__)
+
+if TYPE_CHECKING:
+  import googleapiclient
+
+
+class GoogleCloudStorageTransfer:
+  """Class to call Google Cloud Storage Transfer APIs.
+
+  Attributes:
+    gcst_api_client: Client to interact with GCST APIs.
```

Comment:
Add types `gcst_api_client (type): blabla`

Reply:
No other class docstring has this ... ?

Reply:
I see it in a few places, at least in AWS/Azure packages. This might not be consistent everywhere, just ignore it for now